clang/lib/Basic/Targets/AArch64.h

```diff
@@ -50,6 +50,7 @@ class LLVM_LIBRARY_VISIBILITY AArch64TargetInfo : public TargetInfo {
   bool HasMatMul = false;
   bool HasBFloat16 = false;
   bool HasSVE2 = false;
+  bool HasSVE2p1 = false;
```

Comment:
FYI
```
/home/david.spickett/llvm-project/clang/lib/Basic/Targets/AArch64.h:53:8: warning: private field 'HasSVE2p1' is not used [-Wunused-private-field]
  bool HasSVE2p1 = false;
       ^
1 warning generated.
```

Reply:
Thanks I saw this too. Testing the fix as we speak.